firebase-database-behavior.html

```diff
@@ -90,9 +90,10 @@
       },
 
       __pathChanged: function(path, oldPath) {
-        if (oldPath != null && !this.disabled && this.__pathReady(path)) {
+        if (!this.disabled && !this.valueIsEmpty(this.data)) {
```

Comment:
we also want to handle the case where `path` is not a valid one,  but `oldPath` was.